### ISSUE TITLE
Does not function when Foo Gallery is installed.

### ISSUE THREAD

**Issue by bryancusatis** [benjaminprojas/wp-editor#34]
Save button shoes a white rectangle instead of actually saving when Foo Gallery is installed.

**Comment by benjaminprojas** [comment]
Which save button is this? Plugin/theme editors? Post editor?

Thabks!

**Comment by bryancusatis** [comment]
I noticed it on the Theme Editor and the Plugin Editor.

**Comment by benjaminprojas** [comment]
Just confirming this is the plugin? https://wordpress.org/plugins/foogallery/

**Comment by bryancusatis** [comment]
Yeah. That's the one. Let me know if you need any additional information.
